### PR TITLE
define the model serve process

### PR DESCRIPTION
**Pull request by cooktheryan**
Model serving process

### CODE REVIEW

**Comment by cooktheryan** [comment]
closes #5

File: kubernetes_yaml/model_serve/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Model Serving
+To serve a model a data connetion to an object store is required. Values must be base64 encoded when using the s3.yaml file that is included.
```
Comment (hemajv) [inline kubernetes_yaml/model_serve/README.md:2]: small nit, should be `connection`

File: kubernetes_yaml/model_serve/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+echo "AWSACCESSKEYSECETVALUE" | base64
+```
+
+Once all values that relate to the object storage are defined create the secret and define the specific namespace to deploy the secret.
+
+```
+oc create -f s3.yaml -n $NAMESPACE
+```
+
+Once the secret has been defined the model serve can use the secreet.
```
Comment (hemajv) [inline kubernetes_yaml/model_serve/README.md:18]: small nit, should be `secret`

**Review by hemajv — COMMENTED** [review]
Thanks for adding the documentation for this @cooktheryan! Just a few minor nit bits, else /lgtm 👍

**Comment by hemajv** [comment]
@cooktheryan once I'm able to test out the mixtral model I will add to this documentation

**Pull request merged**